libs/docs/platform/table/examples/virtual-scroll/platform-table-virtual-scroll-example.component.ts

```diff
@@ -16,12 +17,38 @@ import {
     changeDetection: ChangeDetectionStrategy.OnPush,
     encapsulation: ViewEncapsulation.None
 })
-export class PlatformTableVirtualScrollExampleComponent {
+export class PlatformTableVirtualScrollExampleComponent implements OnInit {
     source: TableDataSource<ExampleItem>;
+    state: TableState;
 
     constructor() {
         this.source = new TableDataSource(new TableDataProviderExample());
     }
+
+    ngOnInit(): void {
+        this.state = this._getDefaultState();
+    }
+
+    _getDefaultState(): TableState {
+        return {
+            columnKeys: [],
+            sortBy: [],
+            filterBy: [],
+            groupBy: [],
+            columns: [],
+            searchInput: {
+                category: null,
+                text: ''
+            },
+            freezeToColumn: null,
+            freezeToEndColumn: null,
+            page: {
+                pageSize: 0,
+                currentPage: 1
+            },
+            scrollTopPosition: 3310
```

Comment:
how do you determine this value?

Reply:
arbitrary for example purposes i am guessing

Reply:
This is just for the exampla to showcase that, you can pre-initialize a table with some initial scroll position that you might have saved. 
The value was taken from this specific example, where I wanted this table to scroll to.